cmd/edgectl/aes_install.go

```diff
@@ -940,6 +962,101 @@ func (i *Installer) Report(eventName string, meta ...ScoutMeta) {
 	}
 }
 
+func (i *Installer) generateCrashReport(sourceError error) {
+	// TODO: Use the live endpoint
+	//reportURL := "https://metriton.datawire.io/crash-report"
```

Comment:
TODO

Reply:
Can you put this and all the other new functions in `install_crash.go` or something like that?